React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -558,7 +558,13 @@ - (NSObject *)accessibilityElement
   NSMutableString *result = [NSMutableString stringWithString:@""];
   for (UIView *subview in view.subviews) {
     NSString *label = subview.accessibilityLabel;
-    if (!label) {
+    static Class otherClass = nil;
+    static BOOL otherClassLookedUp = NO;
+    if (!otherClass && !otherClassLookedUp) {
```

Comment:
@mrmacete Is this intentional to be true always? I'm a bit confused on this, why you add this three (3) lines 561..563.

Reply:
The `static` variables are initialised only once, then on subsequent invocations of the same function they retain their value (like as if they were global variables, but scoped only to the function / block they belong to). 

It's intended as an optimisation to avoid calling `NSClassFromString` all the time.

Reply:
Got it, I think you can directly `![subview isKindOfClass:<>]`, removing line 561 to 566. Or not possible?

Reply:
dynamic lookup is safer and less noisy, because `RCTViewComponentView` and `RCTView` apparently don't "know" each other at compile time and they **may** not even be both present at the same time in any given compiled app.

Reply:
(and i think this is the fundamental reason why this code is duped)